components/KiboShippingOptions.vue

```diff
@@ -0,0 +1,63 @@
+<template>
```

Comment:
change "shipping" to "fulfillment" across this entire component.

Reply:
renamed.